mathesar_ui/src/stores/tables.ts

```diff
@@ -416,3 +417,9 @@ export const currentTable = derived(
       ? undefined
       : $tables.data.get($currentTableId),
 );
+
+export function getJoinableTablesResult(tableId: number) {
+  return getAPI<JoinableTablesResult>(
+    `/api/db/v0/tables/${tableId}/joinable_tables/?max_depth=1`,
```

Comment:
`max_depth` should be configurable and must be accepted as a function parameter, instead of being hardcoded as 1.

Reply:
c4f7eb96

Reply:
c4f7eb96